src/airtime.cpp

```diff
@@ -6,20 +6,24 @@ AirTime *airTime = NULL;
 
 // Don't read out of this directly. Use the helper functions.
 
+#define MAX_PERIODS 8
```

Comment:
Maybe better to use `PERIODS_TO_LOG` here, which is already defined in the header file.